apps/console/src/features/identity-providers/components/settings/connected-apps.tsx

```diff
@@ -0,0 +1,529 @@
+    /**
+     * Default list item limit.
+     */
+    defaultListItemLimit?: number;
+     /**
+     * Is the connected apps info request loading.
+     */
+    isLoading?: boolean;
+    /**
+     * Is the list rendered on a portal.
+     */
+    isRenderedOnPortal?: boolean;
+    /**
+     * Specifies if the component should only be read-only.
+     */
+    isReadOnly: boolean;
+    /**
+     * Loading Component.
+     */
+    loader: () => ReactElement;
+}
+
+/**
+ *  Connected Apps settings component.
+ *
+ * @param props - Props injected to the component.
+ * @returns ReactElement
+ */
+export const ConnectedApps: FunctionComponent<ConnectedAppsPropsInterface> = (
+    props: ConnectedAppsPropsInterface
+): ReactElement => {
+
+    const {
+        editingIDP,
+        isSetStrongerAuth,
+        defaultListItemLimit,
+        showListItemActions,
+        onListItemClick,
+        selection,
+        isRenderedOnPortal,
+        isLoading,
+        loader: Loader,
+        [ "data-testid" ]: testId
+    } = props;
+    
+    const dispatch = useDispatch();
+
+    const [ connectedApps, setConnectedApps ] = useState<ConnectedAppInterface[]>();
+    const [ filterSelectedApps, setFilterSelectedApps ] = useState<ConnectedAppInterface[]>([]);
+    const [ connectedAppsCount, setconnectedAppsCount ] = useState<number>(0);
+    const [ isAppsLoading, setIsAppsLoading ] = useState<boolean>(true);
+    const [ searchQuery, setSearchQuery ] = useState<string>("");
+    const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state.config.ui.features);
+    const applicationTemplates: ApplicationTemplateListItemInterface[] = useSelector(
+        (state: AppState) => state.application.templates);
+    const groupedApplicationTemplates: ApplicationTemplateListItemInterface[] = useSelector(
+        (state: AppState) => state?.application?.groupedTemplates);
+    const [
+        isApplicationTemplateRequestLoading,
+        setApplicationTemplateRequestLoadingStatus
+    ] = useState<boolean>(false);
+
+    const { t } = useTranslation();
+
+    useEffect(() => {
+        setIsAppsLoading(true);
+        getIDPConnectedApps(editingIDP.id)
+            .then(async (response: ConnectedAppsInterface) => {
+                setconnectedAppsCount(response.count);
+                
+                if (response.count > 0) {
+                    
+                    const appRequests: Promise<any>[] = response.connectedApps.map((app: ConnectedAppInterface) => {
+                        return getApplicationDetails(app.appId);
+                    });
+
+                    const results: ApplicationBasicInterface[] = await Promise.all(
+                        appRequests.map(response => response.catch(error => {
+                            dispatch(addAlert({
+                                description: error?.description
+                                || "Error occurred while trying to retrieve connected applications.",
```

Comment:
Let's move these strings to i18n. Check in other places as well

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3494/commits/ce8c570994579ea30e3ab52031945bf056094375